src/main/java/com/antkorwin/betterstrings/BetterStringsProcessor.java

```diff
@@ -70,12 +70,7 @@ public SourceVersion getSupportedSourceVersion() {
 	}
 
 	private void printBanner() {
-		String banner = "\n" +
-		                "___  ____ ___ ___ ____ ____    ____ ___ ____ _ _  _ ____ ____ \n" +
-		                "|__] |___  |   |  |___ |__/    [__   |  |__/ | |\\ | | __ [__  \n" +
-		                "|__] |___  |   |  |___ |  \\    ___]  |  |  \\ | | \\| |__] ___] \n" +
-		                "   v0.1 String Interpolation Java Plugin, by Anatoliy Korovin  ";
-
+		String banner = "v0.3 String Interpolation Java Plugin, by Anatoliy Korovin";
```

Comment:
Hmmm I'd rather advice to remove any text completely so as to not pollute Gradle output

Reply:
Esp. in project with more subprojects that's getting pretty.... redundant